src/main/kotlin/io/provenance/client/gas/prices/UrlGasPrices.kt

```diff
@@ -0,0 +1,35 @@
+package io.provenance.client.gas.prices
+
+import com.google.gson.Gson
+import cosmos.base.v1beta1.CoinOuterClass
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.impl.client.HttpClientBuilder
+import java.io.InputStreamReader
+
+/**
+ * When provided with a url, fetches an object of shape '{"gasPrice":nnn,"gasPriceDenom":"denom"}'
+ */
+open class UrlGasPrices(private val uri: String) : GasPrices {
+    private val client = HttpClientBuilder.create().build()
+    private val gson = Gson().newBuilder().create()
```

Comment:
no jackson 😹

Reply:
jackson wasn't in the current deps. gson was. 🤷